code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -496,6 +511,12 @@
 			return 0
 	return 1
 
+/mob/living/simple_animal/proc/attack_threshold_check(damage, damagetype = BRUTE)
+	if(damage <= force_threshold || ignored_damage_types[damagetype])
+		visible_message("<span class='warning'>[src] looks unharmed.</span>")
+	else
+		adjustBruteLoss(damage)

```

Comment:
You may want to make this proc work for all damage types, as currently it'll apply brute damage no matter what damage type you use, assuming the mob isn't immune to brute. Also, the ignored_damage_types check is both here and in the adjust[Damage Type]Loss procs as well, meaning it's kind of redundant to have.